python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/metadata.schema.yml

```
@@ -34,6 +34,45 @@ keys:
         type: str
         convert_types:
           - int
+      hardware_requirements:
+        type: dict
+        description: |-
+          Defines the minimum hardware specifications required for the device.
+          This section is used for validation and documentation purposes, primarily by the `anta_runner` role.
+        keys:
+          min_power_supplies:
+            type: int
+            description: |-
+              Minimum number of power supplies required for the device.
+              Used for documentation and validation by the `anta_runner` role.
+          min_fans:
+            type: int
+            description: |-
+              Minimum number of fans required for the device.
+              Used for documentation and validation by the `anta_runner` role.
+          min_supervisors:
+            type: int
+            description: |-
+              Minimum number of supervisor modules required for the device.
+              Used for documentation and validation by the `anta_runner` role.
+          min_line_cards:
+            type: int
+            description: |-
+              Minimum number of line cards required for the device.
+              Used for documentation and validation by the `anta_runner` role.
+          min_fabric_cards:
+            type: int
+            description: |-
+              Minimum number of fabric cards required for the device.
+              Used for documentation and validation by the `anta_runner` role.
+          transceiver_manufacturers:
+            type: list
+            description: |-
+              Expected list of approved transceiver manufacturers for the device.
+              For all platforms, it would be populated with ['Arastra, Inc.', 'Arista Networks'] by default.
```

We don't know if people are using `eos_designs` si there is no default.

```suggestion
            description: |-
              Expected list of approved transceiver manufacturers for the device.
```

---

updated